pkg/minishift/openshift/service.go

```diff
@@ -86,12 +103,20 @@ func GetServiceURLs(serviceListNamespace string) ([]ServiceURL, error) {
 
 	// iterate over namespaces, get command output, format route in ServiceURL
 	for _, namespace := range namespaces {
-		outputData, err := getServiceURLsOutput(namespace)
+		serviceNodePort, err := getService(namespace, ip)
+		if err != nil {
+			fmt.Println("Here is errror")
+			return serviceURLs, err
+		}
+		outputData, err := getRouteURLsOutput(namespace)
```

Comment:
@hferentschik It actually not the routes url but contain string output which have route name and it's url so I think instead `outputData` I will change it to `routeSpec` and `getRouteSpec`.